src/content/deployment/subctl.en.md

```diff
@@ -0,0 +1,190 @@
+---
+title: "subctl"
+weight: 10
+---
+
+`subctl` is a commandline utility designed to simplify the deployment and maintenance of
+submariner across your your clusters.
+
+### Synopsis
+
+`subctl [command] [--flags] ...`
+
+### Description
+
+The subctl program works with the [submariner-operator](https://github.com/submariner-io/submariner-operator)
+it helps automate the operator deployment process reducing the possibility of mistakes during
+the process.
+
+Subctl connects to your cluster(s) and performs the requested *command* operation. For the `join`
+operation it will start by deploying the submariner-operator.
+
+### Commands
+
+#### deploy-broker
+
+`subctl deploy-broker [flags]`
+
+The **deploy-broker** command will configure the cluster pointed out by the `--kubeconfig`
+(or `KUBECONFIG` env var) and the `--kubecontext` flag as the broker. It will install
+the necessary CRDs and the  `submariner-k8s-broker` namespace.
+
+It will generate the `broker-info.subm` file which can be used with the `join` command
+to connect clusters to the broker. This file contains the following details:
+
+ * Encryption PSK key
+ * Broker access details for subsequent `subctl` runs.
+ * Service discovery settings
+ * Globalnet settings
+
+##### deploy-broker flags
+
+    -h, --help                           help for deploy-broker
+
+    --kubeconfig string                  absolute path(s) to the kubeconfig file(s) (default "$HOME/.kube/config")
+
+    --kubecontext string                 kubeconfig context to use
+
+    --service-discovery                  Enable Multi Cluster Service Discovery
+
+    --globalnet                          Enable support for Overlapping CIDRs in connecting clusters (default disabled)
```

Comment:
Let's do it, but it's just out of subctl -h ... may be we should also send a patch there :)

Reply:
That's the problem of copying the output verbatim into the guides.. I find it kind of pointless to play catch up, just document the important flags and anything else can be figured out on the command line.

Either this or have some auto generated docs from the flags for these sections since updating them manually will be a PITA and they're bound to diverge...